server/swissvoice/static/css/style.css

```diff
@@ -81,3 +112,15 @@ hr.hz-line {
         font-size: 1.2rem;
     }
 }
+
+
+/*--------------------------------------------------*/
+/*medis querry for background-image*/
```

Comment:
media query?

Reply:
Is needed so the bubble doesn't cover up the titel (only the case for smaller screens and height dependent)